tests/terraform/parser/resources/parser_scenarios/merge_function/main.tf

```diff
@@ -0,0 +1,33 @@
+locals {
+  common_tags = {
+    Tag1 = "one"
+    Tag2 = "two"
+  }
+  common_data_tags = {
+    Tag3 = "three"
+  }
+
+  local_to_local = merge(local.common_tags, local.common_data_tags)
+  local_to_manual = merge(local.common_tags, {Tag4 = "four"})
+  manual_to_local = merge({Tag4 = "four"}, local.common_tags)
+  manual_to_manual = merge({Tag4 = "four"}, {Tag5 = "five"})
```

Comment:
@robeden I've seen many cases where the merge is not a oneliner, i.e.:
```terraform
local_to_manual = merge(
    local.common_tags,
    {Tag4 = "four"},
    {Tag2="Dev"}
)
```

Can you add a test case to ensure this is working properly?

Reply:
Good call because the HCL2 parser returns some [weird results](https://github.com/bridgecrewio/checkov/pull/771/commits/44b5dd3e8119b3b2e8340acbff0e5b964b457d53#diff-23faf3382230bc5b99ec03b5e221636a628de63406746fe5063c5c5aa80ec11fR40) when that happens. It handles it now.